docs/INSTALLATION.md

```diff
@@ -12,6 +12,24 @@ stack --version
 If it outputs a Stack version equal to or greater than 1.1.2, then you have it
 installed correctly.
 
+## Installing GHC
+
+Stack will automatically download and install GHC (the Glasgow Haskell
+compiler) when it is required. You can start the installation test it
```

Comment:
in "...start the installation test it by running..." I think the sentence is more grammatically correct if "it" is removed

Reply:
Yes, that's an error ☺